config/schema/format_strawberryfield.schema.yml

```diff
@@ -33,6 +33,8 @@ field.formatter.settings.strawberry_audio_formatter:
       type: string
     number_media:
```

Comment:
Which one is right here number_media or number_audios? You can not keep both

Reply:
@DiegoPino OK makes sense, I didn't want to delete anything if both were used. Will delete.

Reply:
@marlo-longley its part of this pulls job, means you should add that to your workflow, to check if the actual values are being used in the formatter configuration forms and defaults. Please check.